packages/setup/src/index.ts

```diff
@@ -1,41 +1,17 @@
 import inquirer from 'inquirer';
-import * as fs from 'fs/promises';
 import * as dotenv from 'dotenv';
+import * as fs from 'fs/promises';
 import { exit } from 'process';
 import os from 'os';
 
 /**
- * Load ENV, templates first, then override with actual ENV values
+ * Load ENV, template first, then override with actual ENV values
  * if there are any.
  */
 
 // Top level
-dotenv.config({ path: '/usr/praise/.env.template', override: true });
-dotenv.config({ path: '/usr/praise/.env', override: true });
-
-// API
-dotenv.config({
-  path: '/usr/praise/packages/api/.env.template',
-  override: true,
-});
-dotenv.config({ path: '/usr/praise/packages/api/.env', override: true });
-
-// Discord Bot
-dotenv.config({
-  path: '/usr/praise/packages/discord-bot/.env.template',
-  override: true,
-});
-dotenv.config({
-  path: '/usr/praise/packages/discord-bot/.env',
-  override: true,
-});
-
-// Frontend
-dotenv.config({
-  path: '/usr/praise/packages/frontend/.env.template',
-  override: true,
-});
-dotenv.config({ path: '/usr/praise/packages/frontend/.env', override: true });
+dotenv.config({ path: '/usr/praise/.env.template', override: false });
+dotenv.config({ path: '/usr/praise/.env', override: false });
```

Comment:
Override should be `true`, right? The template should override any pre-existing env settings. And the "real" `.env` should override the template.